help/en/releasenotes/current-draft-note.shtml

```diff
@@ -392,6 +395,14 @@
                         Example definitions are LokSound 5(uses F0-F31) and LokSound V3.5(uses only F0-F15).
                     </li>
                 </ul>
+            <p>
```

Comment:
Should we be putting `<p>` after `</ul>` inside a list element? I usually see that going straight to the end of the enclosing `<li>`.  Not sure of the HTML rules here...

Reply:
I _think_ this will add a chunk of extra white space. Could argue that it's superfluous.

Reply:
@bobjacobsen It's the only place I could find where NetBeans 11.3 didn't complain about HTML style rules when I inserted it.

@mattharris I put it in because without it my Safari and Chrome both rendered it like this:
![without p](https://user-images.githubusercontent.com/13841457/90820531-de174580-e374-11ea-96a7-8c305c51b25c.png)
I found that confusing (the grouping appeared wrong to my eyes).

With the `<p>` put where I did, the section renders like this:
![with p](https://user-images.githubusercontent.com/13841457/90820808-4e25cb80-e375-11ea-8ec5-b194f0737624.png)

The second looks better to my eyes.